jest.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
         '/packages/create-react-admin/templates',
     ],
     transformIgnorePatterns: [
-        '[/\\\\]node_modules[/\\\\](?!(@hookform)/).+\\.(js|jsx|mjs|ts|tsx)$',
+        '[/\\\\]node_modules[/\\\\](?!(@hookform|query-string|decode-uri-component|filter-obj|split-on-first)/).+\\.(js|jsx|mjs|ts|tsx)$',
```

Comment:
does that mean that react-admin users will have to do the same in their jest configuration?

Reply:
No, this is specific to Jest

Reply:
This is exactly my question